pkg/telemetry/prometheus/packets.go

```diff
@@ -178,6 +193,26 @@ func initPacketStats(nodeID string, nodeType livekit.NodeType) {
 	promPacketBytesIncomingRetransmit = promPacketBytes.WithLabelValues(string(Incoming), transmissionRetransmit)
 	promPacketBytesOutgoingInitial = promPacketBytes.WithLabelValues(string(Outgoing), transmissionInitial)
 	promPacketBytesOutgoingRetransmit = promPacketBytes.WithLabelValues(string(Outgoing), transmissionRetransmit)
+
+	var err error
+	asnReader, err = geoip2.Open("/opt/maxmind/geoip.db")
+	if err != nil {
+		fmt.Println("***** FAILED TO READ GEO IP DB:", err)
```

Comment:
use Logger that Livekit already has.

Reply:
no Logger is available in this module.

Reply:
use the global `Logger` in `logger` package.